src/raster/r.survey/README.md

```diff
@@ -0,0 +1,272 @@
+# r.survey.py
+
+
+<html>
```

Comment:
@IvanMarchesini also this file is superfluous (and not even MD) - we have src/raster/r.survey/r.survey.html

Reply:
Thank you. Done